ts/features/itwallet/walletInstance/store/reducers/index.ts

```diff
@@ -1,21 +1,31 @@
-import * as O from "fp-ts/lib/Option";
-import { flow } from "fp-ts/lib/function";
 import { PersistConfig, persistReducer } from "redux-persist";
-import { createSelector } from "reselect";
 import { getType } from "typesafe-actions";
 import { Action } from "../../../../../store/actions/types";
-import { GlobalState } from "../../../../../store/reducers/types";
 import itwCreateSecureStorage from "../../../common/store/storages/itwSecureStorage";
-import { isWalletInstanceAttestationValid } from "../../../common/utils/itwAttestationUtils";
 import { itwLifecycleStoresReset } from "../../../lifecycle/store/actions";
-import { itwWalletInstanceAttestationStore } from "../actions";
+import {
+  itwWalletInstanceAttestationStore,
+  itwUpdateWalletInstanceStatus,
+  itwWalletInstanceSetAlertShown
+} from "../actions";
+import { WalletInstanceRevocationReason } from "../../../common/utils/itwTypesUtils";
 
 export type ItwWalletInstanceState = {
   attestation: string | undefined;
+  revocation: {
+    isRevoked: boolean;
+    alertShown: boolean;
+    revocationReason?: WalletInstanceRevocationReason;
+  };
```

Comment:
```suggestion
  status: WalletInstanceStatus | undefined;
  revocationAlertShown: boolean | undefined;
```

I would keep the information separated. What do you think?
Also, I don't think we need to persist this information. Do we?

Reply:
Yes, it makes sense to keep that information separated.
Do you mean if we need to persist `revocationAlertShown`?

Reply:
I mean the `status` data. Since `status` is something related to the current WI, which is being removed for the reason that it is revoked, we need the its `status` information only for the current session. So we can avoid to persist it.